src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -285,6 +285,10 @@
      *                              A CredentialsLoader object created using the
      *                              Google\Auth library.
      * }
+     @if xapiClass.hasMissingDefaultOptions
+     * @throws ValidationException Throws a ValidationException if required arguments are missing
+     *                             from the $options array.
+     @end
```

Comment:
The stars should be indented here (for the snippet's indentation - won't affect the output indentation)